### DISCUSSION TITLE
Creating NetCDF files

### DISCUSSION THREAD

**Discussion by PendleWitch** [gnudatalanguage/gdl#950] [Q&A]
Hello,
I'm looking to write a quick pro to make a NetCDF file. 
It seems that ncdf_create isn't set up in gdl. Is that right?
Many thanks.

**Comment by slayoo** [comment]
Hello,

`NCDF_CREATE` is implemented in GDL. If it is not available on your machine it likely means that netCDF support was switched off during compilation of GDL, to enable it use `-DNETCDF=ON` as a flag for CMake when building GDL.
Some example GDL code using the built-in netCDF routines can be found e.g. here: https://github.com/gnudatalanguage/gdl/blob/master/testsuite/nc_test.pro

HTH,
Sylwester

**Reply by PendleWitch** [reply]
Hello Sylwester,
Thank you for the reply. I did see that command on  a search, but since i used the gdl installer to install, then I wasn't sure how it would work with that.
I will have a try soon with your suggestions. Thanks again!

**Comment by slayoo** [comment]
@pjb7687 Could you confirm how to get the GDL installer with netCDF enabled?
Also, BTW it seems that currently the installer artefacts are only produced for Debug builds which probably will result in users complaining about performance! :)
Thanks,
S.

**Reply by pjb7687** [reply]
Hi @slayoo, Yes the netCDF is enabled with Windows builds. Regarding the installer, I will reply to #954...
